src/ui/qgsmapcanvasdockwidgetbase.ui

```diff
@@ -151,6 +151,17 @@
     <string>Show Labels</string>
    </property>
   </action>
+  <action name="mActionAutoZoomToSelection">
```

Comment:
I think this should be moved up to sit below "syncronize view center with main map", and these two controls made radio buttons instead of checkboxes (since they are mutually exclusive)

Reply:
Yep good point.